### ISSUE TITLE
Launch instance opens wrong page

### ISSUE THREAD

**Issue by chris-miner** [microsoft/vscode-edge-devtools#889]
**Environment (please complete the following information):**
 - OS: macOS 12.1
 - Extension version: v1.4.5

**Describe the bug:**

I clicked on launch project in the extension view, and a page that has nothing to do with my project is opened.

**Repro steps:**

1. created a new project with a basic html file.  body contained only an h1 tag.  
2. with the html file active, clicked on launch project in the dev tools view.
3. two tabs are opened.  one with the dev tools and one with dev tools screen cast
4. the screen cast tab shows contents of file:///Users/chris/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-1.4.5/out/startpage/index.html

**Expected behavior:**

I expected the screen cast tab to show a page related to my project.  (e.g. my only html file titled index.html)

**Additional context:**

I closed the  instance listed under targets in the edge view and clicked on launch instance again after selecting the index.html for a second time.  This time the screen cast tab opened for "about:blank".  Also unexpected.

**Comment by chris-miner** [comment]
After reading more documentation, it looks like this is the way it is supposed to work.  Personally, I'd rather it loaded-up the html page I'm currently editing rather than having to create the launch.json file and then edit the url/file entry.  So I guess this would be a feature request related to a better initial use experience.

**Comment by chris-miner** [comment]
> you want to edit a single file rather than run a local server. As any file is most likely to have other dependencies (styles, scripts and so on)

We'll, I expected to see my file in the browser is the key thing. That's not unreasonable.

Besides, while I'd normally use a web-server, it isn't strictly speaking necessary.

If I open a file directly off the file system the browser will happily pull in referenced resources even without the support of a (web)server.

**Comment by captainbrosset** [comment]
I tend to agree with @chris-miner here. Very often when I start a new web project, I start with a simple index.html file and no web server. I put in my content, and quickly after I need devtools to start styling the page.

When using the extension, the first-run experience in that workflow could be a bit better. I think it would be fair for the extension to detect the main index.html file and load it in the browser using the `file://` protocol.
